### ISSUE TITLE
Falsely reporting "Use snake_case for source file names"

### ISSUE THREAD

**Issue by joshminnie** [AtomLinter/linter-rubocop#46]
When running Rubocop from the command line I don't receive errors, however, on pretty much every ruby file I am consistently getting the Rubocop error `Use snake_case for source file names`.

Sample ruby file that demonstrates the error

```ruby
class Plan < ActiveRecord::Base
end
```

Screenshot showing the error

![false-positive](https://cloud.githubusercontent.com/assets/117469/8340692/1b278df0-1a8e-11e5-808a-d567a8abde17.png)

Using:

- Atom v. 0.211.0
- linter v. 1.0.2
- linter-rubocop v. 0.2.7

**Comment by AsaAyers** [comment]
This will be fixed in the next update of `linter`. See https://github.com/AtomLinter/Linter/issues/585

**Comment by AsaAyers** [comment]
I just published linter@1.0.3, which fixes this

**Comment by joshminnie** [comment]
Awesome! Thanks for the quick update.
